### PR TITLE
1_added requirements and readme

### LINKED ISSUE #1: Add conda environment

### PR DESCRIPTION
**Pull request by Juan-Mateos**
Closes #1

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+altair
+jupyterlab
+matplotlib
+numpy
+panda
```
Comment (Jack-0-0) [inline requirements.txt:5]: missing an 's' from pandas

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+scipy
+sklearn
```
Comment (Jack-0-0) [inline requirements.txt:7]: i think sklearn needs to be scikit-learn

File: conda_environment.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+name: causal_ml_learning
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - pip
+  - python=3.8
```
Comment (georgerichardson) [inline conda_environment.yaml:7]:
```suggestion
  - python=3.8

```

**Review by georgerichardson — APPROVED** [review]
* Ran `conda env create -f conda_environment.yaml`
* Ran `conda activate causal_ml_learning`
* Ran `pip install -r requirements.txt`
* Started `ipython` and tested `import econml, statsmodels, dowhy`

**Review by Jack-0-0 — APPROVED** [review]
It sets up and runs fine for me 👍

**Pull request merged**